Signal/src/network/PushManager.m

```diff
@@ -162,6 +162,10 @@ - (void)application:(UIApplication *)application didReceiveRemoteNotification:(N
     }
 }
 
+- (void)applicationDidBecomeActive {
```

Comment:
There is no consistent style across the codebase, but I'm trying to converge on one by using `git clang-format`. You might consider adding it to your git hooks.